### PR TITLE
add doc for accelerating initrd/rootimg generation with pigz

### PR DESCRIPTION
**Pull request by immarvin**
The html doc can be found in
http://10.3.1.3/git/doc-html/doc-pigz/build/html/guides/admin-guides/manage_clusters/ppc64le/diskless/customize_image/acc_initrd_rootimg_gen_ppc64le.html

the previous pull request https://github.com/xcat2/xcat-core/pull/133

### CODE REVIEW

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/acc_initrd_rootimg_gen.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+========================================================
+
+Generating diskless initrd with ``genimage`` and compressed rootimg with ``packimage`` and ``liteimg`` is a time-comsuming process, it can be accelerated by enabling paralell compression tool ``pigz`` on the management node with multiple processors and cores.
+
+The paralell compression tool ``pigz`` can be enabled by installing ``pigz`` package on the management server or diskless rootimg. Depending on the method of generating the initrd and compressed rootimg, the steps differ in different Linux distributions. 
+
+
+* Enabling the ``pigz`` on :ref:`Ubuntu Server LTS<ubuntu-os-support-label>`
+--------------------------------------------------------------------------
+
```
Comment (zet809) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/acc_initrd_rootimg_gen.rst]: Need to include diskless section

Comment (immarvin) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/acc_initrd_rootimg_gen.rst]: This section is for generating initrd and rootimg in diskless "genimage" and "packimage".

pigz is shipped in ubuntu and sles, nit shipped in redhat base iso

**Comment by immarvin** [comment]
@whowutwut , I have updated the pull request according to your comment,

**Comment by daniceexi** [comment]
@immarvin 
Please follow the [doc_guidlines](https://xcat-docs.readthedocs.org/en/latest/developers/doc_guidelines.html#add-os-or-arch-specific-contents) to create the branches for OS specific contents.

Also try to indent the content for the content in each list.

**Comment by daniceexi** [comment]
I reviewed the change and agree to check in. And thanks the comments for #133

**Comment by whowutwut** [comment]
@immarvin , nice job on the doc!

**Pull request merged**